packages/@sanity/form-builder/src/inputs/ReferenceInput/ReferenceInput.tsx

```diff
@@ -217,6 +222,15 @@ export const ReferenceInput = forwardRef(function ReferenceInput(
       description={type.description}
     >
       <Stack space={3}>
+        {hasInsufficientPermissions && (
+          <Alert title="Insufficient permissions to access this reference" status="warning">
+            <Text as="p" muted size={1}>
```

Comment:
Are there any particular reason for using `as="p"` here? Curious

Reply:
I was following a previous convention I saw in this document. Unless `as="p"` is the default for Text, I think this has some nice semantics.